com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/model/ParameterMetadata.java

```diff
@@ -58,7 +58,7 @@ public ParameterMetadata build() {
             result.setId(id);
             result.setRequired(CommonUtil.getOrDefault(required, false));
             result.setDefaultValue(defaultValue);
-            result.setType(type);
+            result.setType(CommonUtil.getOrDefault(type, ParameterType.STRING));
```

Comment:
This is done in order to preserve the previous behaviour of the ParameterMetadata builder, where if no type is specified then the type is defaulted to String.